Automattic/sensei
# Fix some JS deprecated functions

### Changes proposed in this Pull Request

* It just fixes some JS deprecated functions.

### Testing instructions


* Create/access a course with at least one lesson.
* Go to the course editor, remove all the blocks, and make sure the legacy meta boxes show up.
* In the bottom meta box, click on "Edit this lesson" and "+ Add Another Lesson", and make sure the respective events are logged properly as before.
* Add the blocks back, and make sure the legacy meta boxes disappear.
* Use the course ordering (`/wp-admin/edit.php?post_type=course&page=course-order`), and make sure it continues working properly.
